tests/test_spark_secrets.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import pytest
+import os
+
+import sdk_cmd
+import sdk_utils
+
+import dcos_utils
+import spark_utils as utils
+
+log = logging.getLogger(__name__)
+auth_token = sdk_cmd.run_cli('config show core.dcos_acs_token', print_output=False).strip()
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_spark(configure_security_spark, configure_universe):
+    try:
+        utils.upload_dcos_test_jar()
+        utils.require_spark()
+        yield
+    finally:
+        utils.teardown_spark()
+
+
+@sdk_utils.dcos_ee_only
+@pytest.mark.sanity
+def test_env_based_ref_secret():
+    secret_path = "/spark/secret-name"
+    secret_value = "secret-value"
+    dcos_utils.delete_secret(secret_path)
+    dcos_utils.create_secret(secret_path, secret_value, False)
+    try:
+        utils.run_tests(
+            app_url=utils.dcos_test_jar_url(),
+            app_args=auth_token,
+            expected_output=secret_value,
+            args=["--conf=spark.mesos.driver.secret.names='"+secret_path+"'",
```

Comment:
let's use `"--conf=spark.mesos.driver.secret.names='{}'".format(secret_path)` instead

Reply:
Changed it to using format method.